### ISSUE TITLE
Unable to create entity with sensuctl create -f

### ISSUE THREAD

**Issue by treydock** [sensu/sensu-go#1564]
## Expected Behavior

I'd expect to be able to create an entity with `sensuctl create -f`

## Current Behavior

```
Error: PUT "/entities/test": {"code":2,"error":"not found"}
```

## Steps to Reproduce (for bugs)

```
[root@sensu-backend ~]# cat /tmp/entity.json
{
  "type": "Entity",
  "spec": {
    "id": "test",
    "class": "agent",
    "deregister": true,
    "keepalive_timeout": 120,
    "organization": "default",
    "environment": "default"
  }
}
[root@sensu-backend ~]# sensuctl create -f /tmp/entity.json
Error: PUT "/entities/test": {"code":2,"error":"not found"}
```

## Context

Automating sensu with @ghoneycutt and have standardize all resource types on `sensuctl create -f`.  Recently appears sensu-go added the ability to create entities so would be handy if `sensuctl create -f` supported the creation of entities.

## Your Environment

* Sensu version used (sensuctl, sensu-backend, and/or sensu-agent): `sensu-ctl version 2.0.0-nightly#9f6fee1, build 9f6fee1926ab1cfc93f03a32e8c1d46a28319732, built 2018-05-18T09:23:16+0000`
* Installation method (packages, binaries, docker etc.): RPM package
* Operating System and version (e.g. Ubuntu 14.04): EL7

**Comment by treydock** [comment]
Possibly relates to #1099 , #1468, #1479

**Comment by echlebek** [comment]
Looks like this was fixed in #1495 \o/
